backend/src/main/java/corea/profile/service/ProfileService.java

```diff
@@ -35,8 +35,8 @@ public ProfileResponse findOne(long memberId) {
     }
 
     private List<String> findTopFeedbackKeywords(long memberId) {
-        List<List<FeedbackKeyword>> reviewerFeedbacks = revieweeToReviewerRepository.findAllKeywordsByReviewerId(memberId);
-        List<List<FeedbackKeyword>> revieweeFeedbacks = reviewerToRevieweeRepository.findAllKeywordsByRevieweeId(memberId);
+        List<List<FeedbackKeyword>> reviewerFeedbacks = socialFeedbackRepository.findAllKeywordsByReviewerId(memberId);
+        List<List<FeedbackKeyword>> revieweeFeedbacks = developFeedbackRepository.findAllKeywordsByRevieweeId(memberId);
```

Comment:
```suggestion
        List<List<FeedbackKeyword>> socialFeedbacks = socialFeedbackRepository.findAllKeywordsByReceiverId(memberId);
        List<List<FeedbackKeyword>> developFeedbacks = developFeedbackRepository.findAllKeywordsByReceiverId(memberId);
```

특별히 남겨둔 이유가 있을까요?
누락이라면 위와 같은 변경을 제안드립니다!